templates/scenarios/sso-tab-js/teamsfx/app.local.yml

```diff
@@ -1,18 +1,58 @@
 version: 1.0.0
 
-# provision:
+registerApp:
+  - uses: env/generate
```

Comment:
I think this driver should be placed to `configureApp`.

Reply:
Removed it.